services/notification-router/src/index.ts

```
@@ -0,0 +1,127 @@
+    slackDelivery,
+    websocketDelivery,
+    baseUrl: process.env.BASE_URL || 'http://localhost:3000',
+    enableAutoRouting: process.env.ENABLE_AUTO_ROUTING !== 'false',
+  });
+
+  // Start router
+  await router.start();
+
+  // Health check endpoint (if running as standalone service)
+  const port = parseInt(process.env.PORT || '3003', 10);
+  const express = require('express');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)